manifests/init.pp

```diff
@@ -12,6 +12,7 @@
   Enum['docker', 'crio'] $container_manager = 'crio',
   String[1] $container_runtime_service      = "${container_manager}.service",
   Optional[String[1]] $crio_package         = undef,
+  String $runc_version                      = 'installed',
```

Comment:
```suggestion
  String[1] $runc_version                   = 'installed',
```

Reply:
next time: pls update also refs in your suggestions 🤣